tests/test_patron.py

```diff
@@ -63,31 +67,36 @@ def test_patron_xml_records_from_dicts_staff_success(staff_patron_all_values_dic
     assert (
         patron_xml_record.user_identifiers.user_identifier.value.string == "222222222"
     )
-
-
-@freeze_time("2023-03-01")
-def test_patron_xml_records_from_dicts_student_success(student_patron_all_values_dict):
-    patron_xml_record = next(
-        patron_xml_records_from_dicts("student", [student_patron_all_values_dict])
-    )
-    assert patron_xml_record.primary_id.string == "STUDENT_KRB_NAME@MIT.EDU"
-    assert patron_xml_record.expiry_date.string == "2023-09-01Z"
-    assert patron_xml_record.purge_date.string == "2025-09-01Z"
     assert (
-        patron_xml_record.emails.email.email_address.string
-        == "STUDENT_KRB_NAME@MIT.EDU"
-    )
-    assert (
-        patron_xml_record.user_identifiers.user_identifier.value.string == "111111111"
+        patron_xml_record.find_all("user_identifier")[1].value.string
```

Comment:
This is kinda ugly and inconsistent but I'm not sure there's a better way to grab the 2nd `user_identifier` value

Reply:
I think it's fine but I would probably get the first `user_identifier` value the same way so it's very clear what's happening here.